### PR TITLE
Fix make test

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

**Comment by gperciva** [comment]
Minimal fix for the setuidgid problem you mentioned in #104, but we might prefer a different solution in this case.  See the discussion there.

File: tests/setuidgid/Makefile.BSD [hunk 1]
```diff
@@ -17,6 +17,6 @@ SRCS	+=	warnp.c
 IDIRS	+=	-I${LIBCPERCIVA_DIR}/util
 
 test:	test_setuidgid
-	@./test_setuidgid.sh
+	@./test_setuidgid $$$$(id -u):$$$$(id -g)
```
Comment (cperciva) [inline tests/setuidgid/Makefile.BSD]: You know that this breaks `make -f Makefile.BSD test`, right?

Comment (gperciva) [reply tests/setuidgid/Makefile.BSD]: ok, I reverted back to using a shell script.  I'll update this PR once #112 is merged (no point in generating more `Makefile` conflicts).

**Comment by gperciva** [comment]
Updated PR

File: tests/setuidgid/Makefile.BSD [hunk 2]
```diff
@@ -17,6 +17,6 @@ SRCS	+=	warnp.c
 IDIRS	+=	-I${LIBCPERCIVA_DIR}/util
 
 test:	test_setuidgid
-	./test_setuidgid.sh
+	./test-setuidgid-minimal.sh
```
Comment (cperciva) [inline tests/setuidgid/Makefile.BSD]: Why not just run the single command here?  Now that we're using awk to extract the test: target there's no issue with multiply escaping `$` characters.

**Pull request merged**